server/models/Checkout.js

```diff
@@ -1,15 +1,16 @@
+const { response } = require('express');
 const mongoose = require('mongoose');
 require('dotenv').config();
 
 const checkoutDBURI = `${process.env.DB_LOCAL}/checkoutDB`;
```

Comment:
might be good to use the mongodb string here. i think we will need to run each others seeding scripts locally. removing the process.env variable will eliminate some potential setup bugs. otherwise i suggest putting some setup steps in your read.me